### ISSUE TITLE
Don't go to file in buffer path on open

### ISSUE THREAD

**Issue by Tloe** [weirongxu/coc-explorer#385]
I normaly start nvim in the folder where I am going to do work.

Now if I have open my vim init file or :CocConfig or any other file outside my starting directory and open coc-explorer it jumps to where the file is located, but I want it to always open where I started nvim from.

**Comment by weirongxu** [comment]
https://github.com/weirongxu/coc-explorer#command
You can use the `root-uri` parameter, or wait for me to implement this feature https://github.com/weirongxu/coc-explorer/issues/198

**Comment by Tloe** [comment]
Right.. I guess that can work for now.. I can have a extra mapping for when I want to go outside the current workingPath..
